src/docfx/template/TemplateEngine.cs

```diff
@@ -98,13 +97,22 @@ public void CopyTo(string outputPath)
 
         public JObject RunJint(string scriptName, JObject model, string methodName = "transform", bool tryParseFromContent = true)
         {
-            var scriptPath = Path.Combine(_templateDir, "ContentTemplate", scriptName);
+            var scriptPath = Path.Combine(_contentTemplateDir, scriptName);
             if (!File.Exists(scriptPath))
             {
                 return model;
             }
 
-            var result = (JObject)_js.Run(scriptPath, methodName, model);
+            var jsResult = _js.Run(scriptPath, methodName, model);
```

Comment:
I'm patching docs-ui here https://dev.azure.com/ceapex/Engineering/_git/docs-ui/pullrequest/13489?_a=overview, could be removed after the patch is deployed.